docs/guides/migrating-from-camunda-platform-7/adjusting-source-code.md

```diff
@@ -0,0 +1,82 @@
+---
+id: adjusting-source-code
+title: Adjusting source code
+description: "Migrate process solutions developed for Camunda Platform 7 to run them on Camunda Platform 8."
+keywords:
+  [
+    Camunda 8,
+    Camunda 7,
+    migration guide,
+    transition,
+    transition guide,
+    Camunda Platform 7,
+  ]
+---
+
+Camunda Platform 8 has a different API than Camunda Platform 7. As a result, you have to migrate some of your code, especially code that does the following:
+
+- Uses the Client API (e.g. to start process instances)
+- Implements [service tasks](/components/modeler/bpmn/service-tasks/service-tasks.md), which can be:
+  - [Java code attached to a service task](https://docs.camunda.org/manual/latest/user-guide/process-engine/delegation-code/) and called by the engine directly (in-VM).
+  - [External tasks](/components/best-practices/development/invoking-services-from-the-process-c7.md#external-tasks), where workers subscribe to the engine.
+
+<!--
+We'll explore these three cases in the sections below.
+
+![spring boot](img/architecture-spring-boot.png)
+-->
+
+For example, to migrate an existing Spring Boot application, take the following steps:
+
+1. Adjust Maven dependencies
+
+- Remove Camunda Platform 7 Spring Boot Starter and all other Camunda dependencies.
+- Add [Spring Zeebe Starter](https://github.com/camunda-community-hub/spring-zeebe).
+
+2. Adjust config
+
+- Make sure to set [Camunda Platform 8 credentials](https://github.com/camunda-community-hub/spring-zeebe#configuring-camunda-platform-8-saas-connection) (for example, in `src/main/resources/application.properties`) and point it to an existing Zeebe cluster.
+- Remove existing Camunda Platform 7 settings.
+
+3. Replace `@EnableProcessApplication` with `@EnableZeebeClient` in your main Spring Boot application class.
```

Comment:
`@EnableZeebeClient` is deprecated in 8.2 and not required anymore.

Reply:
I can remove this line entirely, but just to confirm, you no longer need to replace `@EnableProcessApplication` with anything?

Reply:
Yes.